database/migrations/2023_12_26_174218_add_update_travel_policy_permission.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Spatie\Permission\Models\Permission;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        app()['cache']->forget('spatie.permission.cache');
+
```

Comment:
this line feels unnecessary

Reply:
sorry 😬 I only meant the extra line break, I believe the cache clearing is important here?

Reply:
I'll revert the whole commit. I prefer the whitespace and the style checkers don't have a problem with it :wink:

The cache clear _probably_ is not necessary since it's already cleared before migrations run, but I've been adding it to every permissions migration

Reply:
fair